instance/models/server.py

```diff
@@ -29,6 +29,7 @@
 from django.db import models
 from django.db.models import Q
 from django_extensions.db.models import TimeStampedModel
+from django_extensions.db.fields.json import JSONField
```

Comment:
Thanks for the tip, @bradenmacdonald !  Also fixed in [8f17ecf](https://github.com/open-craft/opencraft/pull/169/commits/8f17ecf57f20fa4e4a0641d4c2746cac5a6ffefd#diff-b66ef4710ed40f1b980e3ab3a34ec812R26).